### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Added docker build details 

## Why?

Visibility and info into the Docker images build process
Plus https://github.com/temporalio/temporal/pull/2456#discussion_r799009240 

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,48 @@
 
 [![Update Submodules](https://github.com/temporalio/docker-builds/actions/workflows/update-submodules.yml/badge.svg)](https://github.com/temporalio/docker-builds/actions/workflows/update-submodules.yml)
 
-# Docker images build and publish
+# Build docker image
 
-A set of pipelines that build:
+Temporal builds 3 Docker images with [every release](https://github.com/temporalio/temporal/releases): Server, Server [with Auto Setup](https://docs.temporal.io/blog/auto-setup), and Admin-Tools. Docker images are built via [Github Actions docker-builds](https://github.com/temporalio/docker-builds/actions) There are [other builds e.g. for CI](https://hub.docker.com/u/temporalio) as well.
```
Comment (alexshtin) [inline README.md:9]: It is this repo now, right?
```suggestion
Temporal builds 3 Docker images with [every release](https://github.com/temporalio/temporal/releases): Server, Server [with Auto Setup](https://docs.temporal.io/blog/auto-setup), and Admin-Tools. Docker images are built via [Github Actions docker-builds](actions) There are [other builds e.g. for CI](https://hub.docker.com/u/temporalio) as well.
```

File: README.md [hunk 2]
```diff
@@ -4,19 +4,48 @@
...
+To build docker image:
+  * [docker](https://docs.docker.com/engine/installation/)
+
+To run docker image with dependencies:
+  * [docker-compose](https://docs.docker.com/compose/install/)
 
 ## Build docker image for any commit
 
-Replace **YOUR_TAG** and **YOUR_CHECKOUT_COMMIT** to build manually:
+Refer to [docker-builds](https://github.com/temporalio/docker-builds) to build the images manually.
```
Comment (alexshtin) [inline README.md:21]: I am in `docker-builds` repo.

**Review by alexshtin — COMMENTED** [review]
This is actually an interesting problem. How to build and run docker image from your local changes? This needs to be think through and documented properly, not just copied from main repo.

**Comment by robholland** [comment]
I suggest closing this, I don't think the content is useful.

**Pull request closed without merging**